tests/socket/socket_tests.cpp

```diff
@@ -276,6 +277,10 @@ connection_monitor_test(
     bpf_program* _program = bpf_object__find_program_by_name(object, "connection_monitor");
     REQUIRE(_program != nullptr);
 
+    uint64_t process_id = get_current_pid_tgid();
```

Comment:
nit: do we have a test case where we validate that the process ID retrieved by the BPF program using the helper function matches with the current process ID?

Reply:
The test already checks for that. If you see any missing cases, please let me know.